### DISCUSSION TITLE
Fiscal Development working group notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#67] [Fiscal Development Group]
# Fiscal Development 
In today's OMI [weekly meeting 11](https://github.com/omigroup/OMI/discussions/54) Evo proposed forming a small group for discussing and planning fiscal development for OMI. We have a lot of work to do to establish documentation tactics, potential grants and funders, opportunities for establish best practices with OMI fiscal development. 

If you would like to join this working group please sign up by commenting below :point_down: 

(note to admins: Feel free to re-write my spec here, just getting something out there to collect sign ups from those who expressed interest in participating)

**Comment by fire** [comment]
I want to join the Fiscal Development working group.

**Reply by RedEagleP1** [reply]
I would like to be part.

**Reply by mrmetaverse** [reply]
Welcome @RedEagleP1 :)

**Comment by humbletim** [comment]
@jsealt's excellent meeting notes from 7/21/2021 OMI Week 13 meeting: https://hackmd.io/@mrmetaverse/omi-072121#Fiscal-Development-Group

## Fiscal Development Group 
- We set up issue [#57](https://github.com/omigroup/OMI/issues/57) to plan and form a team to discuss who we can reach out to for grants and resources. 
- We have had some volunteers, and would like to remind the group that anyone else may get involved 
- At present moment the group consists of Evo, Jesse, Robert, and Fire. 
- Ways to fund hackathons, methods of promoting the devs in this community, resources for these types of things. 
- Does anyone have connections to metaverse funds? 
- **:dart: goal:** Identify the top 5 groups to apply
- Our next steps include developing our one pager for folks to be able to begin partnering. If we are a community group asking other orgs to support us, are we asking them to support an event, or are we asking them to support the open metaverse at large (paraphrase). 
- This group will be meeting a little later today 
- More information on [Open Funding OS Resources](https://github.com/ralphtheninja/open-funding)

**Comment by mrmetaverse** [comment]
The Fiscal development working group met last night to continue working. Notes can be found on hackmd [Fiscal Development Meeting 7/21/21](https://hackmd.io/@robertlong/omi-fiscal-development)

## Potential Projects to Fund

- glTF Blender Addon
    - Original funding has ran out
    - Work with Blender Foundation and Khronos to increase funding to this project
- Interoperable Art Day
    - Encourage artists to build glTF environments / avatars / props / experiences that can be used for interoperability experiments, the OMI home space etc.
    - Determine pain points in glTF tooling and get feedback from artists
    - OMI can provide support and community for artists during the event
    - Use funding for awards
- Open Marketplace
    - Icosa: https://beta.icosa.gallery/
    - pmndrs market: https://market.pmnd.rs/

## Next Steps

- Lightweight event ~1 month from now
    - OMI Demo Day
    - Possibly taking place in Jel
        - Pros:
            - High room capacity
            - Screen sharing
        - Cons:
            - Voxelized glTFs for artists
            - Hopping between experiences lets users see them first hand
        - August 19th or August 26th
- Larger event in October
    - Art Day / Hackathon

**Comment by mrmetaverse** [comment]
In the teams meeting yesterday, the group discussed getting started with Demo days #59 
with initial dates set to be either Aug 19th or Aug 26th

**Comment by mrmetaverse** [comment]
converted from issue to discussion to continue working

**Comment by mrmetaverse** [comment]
Just realized we don't have a fiscal development channel on discord. Is this part of the community-development channel or do we want to create a new discord channel? Furthermore, do we want to create a new repo for managing fiscal development backlogs or continue to use the core omigroup/omigroup repo? cc: @humbletim  @LightLodges @fire 

Tomorrow some members of the open metaverse community will announce a new non profit, and there is a new DAO in the works as well (technically more than one, as many OMI participants are creating communities) for sustainably funding the open metaverse. These are separate nodes in the ecosystem, and are not owned by OMI. I think we could benefit form still establishing a true fiscal development group inside of OMI as OMI may want to issue grants, and fund hackathons, and partner on events, etc. just because some of us are working on funds, shouldn't mean OMI can't help too. Let's just stay mindful of scope creep and avoid becoming the everything-org rather than the interoperability-group that we set out to be. 

I will add this to Wednesdays agenda
